frontend/src/styles/app.scss

```diff
@@ -109,7 +109,7 @@ sub {
 }
 
 .separator {
-  border-bottom: 1px solid $transparent-blue;
+  border-bottom: 1px solid adjust-color($primary-light, $alpha: -0.5);
```

Comment:
Should we be changing this to one of the colors in the design scheme instead of changing the color using opacity?

Reply:
Wanted to do that at first. But the design system uses opacity instead, so I followed for consistency. 

![image](https://user-images.githubusercontent.com/3666479/98065924-90e54300-1e90-11eb-9a6c-41cceb14c4bc.png)